cpp/src/com/google/idea/blaze/cpp/BlazeCWorkspace.java

```diff
@@ -165,6 +172,97 @@ public void run(ProgressIndicator indicator) {
             });
   }
 
+  private Path trimStart(Path value, @Nullable Path prefix) {
+    if (prefix == null || !value.startsWith(prefix)) {
+        return value;
+    }
+
+    return value.subpath(prefix.getNameCount(), value.getNameCount());
+  }
+
+  private @Nullable Path pathOf(@Nullable String value) {
+    if (value == null || value.isEmpty()) {
+        return null;
+    }
+
+    try {
+      return Path.of(value);
+    } catch (InvalidPathException e) {
+      return null;
+    }
+  }
+
+  private List<String> collectIncludes(Path root, TargetKey targetKey, BlazeProjectData blazeProjectData) {
+    TargetIdeInfo targetIdeInfo = blazeProjectData.getTargetMap().get(targetKey);
+    if (targetIdeInfo == null || targetIdeInfo.getcIdeInfo() == null) {
+      return Collections.emptyList();
+    }
+
+    CIdeInfo cIdeInfo = targetIdeInfo.getcIdeInfo();
+    Path includePrefix = pathOf(cIdeInfo.getIncludePrefix());
+    Path stripPrefix = pathOf(cIdeInfo.getStripIncludePrefix());
```

Comment:
related to #5969, strip prefix might be `//foo/bar` and this code will break on Windows (yes we do not support it, though it works and maybe we should not introduce new issues).